features/HTTP.feature

```diff
@@ -170,6 +170,15 @@ Feature: Test HTTP API
     Then I create a restricted user "restricteduser1" with id "restricteduser1-id"
 
   @usingHttp @cleanSecurity
+  Scenario: Role mapping
+    Given I get the role schema
```

Comment:
I have the feeling there is ambiguity between "schema" and "mapping". Do we really need both terms?